docs/client.rst

```diff
@@ -286,15 +287,14 @@ This implies:
 
 1. Flashes all output streams sending the data through the transport.
 2. If there is any reliable stream, it will perform the asociated reliable behaviour to ensure the communication.
-3. Listens messages from the agent and call the associated callback (a topic callback or a status callback).
+3. Listens messages from the agent and call the associated callback if exists (a topic callback or a status callback).
 
-The ``_until_timeout`` suffix function version will perform these actions until the waiting for a new message reaches the timeout.
-Only if the time waiting for a message overcome the timeout, the function finished.
-The function will return ``true`` if has been listened a message, ``false`` if the timeout has been reached.
+The ``_until_timeout`` suffix function version will perform these actions until receiving one message.
```

Comment:
Can you specify it finish when a user's data/message is received?

Reply:
Done!